ext/hal/altera/altera_hal/HAL/src/altera_common.c

```diff
@@ -80,3 +80,7 @@ unsigned int alt_busy_sleep (unsigned int us)
 	k_busy_wait(us);
 	return 0;
 }
+
+void alt_printf(const char *fmt, ...)
+{
```

Comment:
what's this function used for? any value in optionally connecting it to printk()?

Reply:
Just want to provide a wrapper as it is being used few places in the HAL driver.

For now, I would like to keep this as dummy function, we can see if it needed in future to compliment with printk.